pkg/process/config/config.go

```diff
@@ -656,3 +686,37 @@ func setupLogger(loggerName config.LoggerName, logFile string, cfg *AgentConfig)
 		config.Datadog.GetBool("log_format_json"),
 	)
 }
+
+func getDDAgentClient() (pb.AgentClient, error) {
```

Comment:
nit: since this seems like a non-config related method, it may be useful to split it out into a different file